Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV PATH "/usr/local/ruby-2.3.3/bin:/opt/.gem/bin:$PATH"
 ENV GEM_HOME "/opt/.gem/"
 RUN echo "gem: --no-document" > /root/.gemrc && \
     gem install \
-      rails:5.0.0 \
+      rails:5.1.7 \
```

Comment:
Why is Rails updated?

Reply:
5.1.x has support for permitting arbitrary hashes.
https://github.com/rails/rails/commit/e86524c0c5a26ceec92895c830d1355ae47a7034